e2etest/get_stats_test.go

```diff
@@ -0,0 +1,142 @@
+package e2etest
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"path"
+	"runtime"
+	"strings"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+	"github.com/datacommonsorg/mixer/server"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc"
+)
+
+// This test runs agains staging staging bt and bq dataset.
+// This is billed to GCP project "datcom-ci"
+// It needs Application Default Credentials to run locally or need to
+// provide service account credential when running on GCP.
+const (
+	btProject   = "google.com:datcom-store-dev"
+	btInstance  = "prophet-cache"
+	bqProject   = "datcom-ci"
```

Comment:
Are we putting the BQ tables now in a different project?

Reply:
This is actually the account the query is billed: https://godoc.org/cloud.google.com/go/bigquery#NewClient